static/app/components/profiling/threadSelector.tsx

```diff
@@ -30,6 +30,7 @@ function ThreadMenuSelector<OptionType extends GeneralSelectValue = GeneralSelec
         ? `tid (${profile.threadId}): ${profile.name}`
         : `tid (${profile.threadId})`,
       value: profile.threadId,
+      disabled: profile.samples.length === 0,
```

Comment:
are we showing a tooltip or something to inform why this thread is disabled?

Reply:
We display 0 samples in the dropdown and I assume it shouldn't be a problem for users to infer that that is why it's disabled. If there is confusion we can revisit later